pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -331,6 +331,10 @@ func (r *ClusterInfoReconciler) getOCPDistributionInfo() (clusterv1beta1.OCPDist
 		return ocpDistributionInfo, client.IgnoreNotFound(err)
 	}
 
+	ocpDistributionInfo.Channel, _, err = unstructured.NestedString(obj.Object, "spec", "channel")
+	if err != nil {
+		klog.Errorf("failed to get OCP channel in clusterVersion: %v", err)
```

Comment:
wait, if the field is not exist what will happen? it is optional

Reply:
the ocpDistributionInfo.Channel will be empty string, be omitempty in json